Kudu.Core/K8SE/K8SEDeploymentHelper.cs

```diff
@@ -140,6 +141,10 @@ public static string GetAppName(HttpContext context)
         public static string GetAppKind(HttpContext context)
         {
             var appKind = context.Request.Headers["K8SE_APP_KIND"].ToString();
+            foreach (var header in context.Request.Headers)
+            {
```

Comment:
do not log as this is customer data

Reply:
This is temporary debugging line.  I'll remove this method before this PR is merged.  We need to test with  kubernetes environment apps. That requires fetch a specific branch. So that I temporary including this.

Reply:
Changes LGTM. If E2E testing is done and OK, please update the PR removing this line and will approve.